develocity-maven.yml

```diff
@@ -80,22 +96,26 @@ spec:
     fi
     if [[ ("${appliedCustomDv}" = "true" || "${appliedCustomCcud}" = "true") && "${enforceUrl}" = 'true' ]]
     then
-      mavenOpts="${mavenOpts} -Dgradle.enterprise.url=${url}"
+      mavenOpts="${mavenOpts} -Dgradle.enterprise.url=${url} -Ddevelocity.url=${url}"
     elif [[ "${appliedCustomDv}" = "false" && "${appliedCustomCcud}" = "false" ]]
     then
-      mavenOpts="${mavenOpts} -Dgradle.enterprise.url=${url}"
+      mavenOpts="${mavenOpts} -Dgradle.enterprise.url=${url} -Ddevelocity.url=${url}"
     fi
     local existingMavenOpts=$(if [ ! -z "$MAVEN_OPTS" ]; then echo "${MAVEN_OPTS} "; else echo ''; fi)
     export MAVEN_OPTS=${existingMavenOpts}${mavenOpts}
   }
-
-  function dvCoordinates() {
-    local coordinates='com.gradle:gradle-enterprise-maven-extension'
+  
+  function detectDvExtension() {
     if [ ! -z "${customMavenExtensionCoordinates}" ]
     then
-      coordinates="${customMavenExtensionCoordinates}"
+      echo "$(detectExtension $1 $customMavenExtensionCoordinates)"
```

Comment:
I would assign a more explicit name to `$1` (`rootDir` ?) to have a bit more context

Reply:
Makes sense 👍 Changed in https://github.com/gradle/develocity-gitlab-templates/pull/22/commits/f4e992203dd742ab0c228696768729c2d2b60215